tables/Tables/BaseTabIter.h

```diff
@@ -104,10 +104,14 @@ class BaseTableIterator
     // Return the next group.
     virtual BaseTable* next();
 
+    // Report Name of slowest column that changes at end of current iteration
+    String keyChange() const { return keyCh_p; };
+
 protected:
     BaseTable*             sortTab_p;     //# Table sorted in iteration order
     uInt                   lastRow_p;     //# last row used from reftab
     uInt                   nrkeys_p;      //# nr of columns in group
+    String                 keyCh_p;       //# the column name ending the current iteration
```

Comment:
By the standards of other var names in this class, this is a quibble.  OK, I'll change it to keyChangeAtLastNext_p.